pandas/errors/__init__.py

```diff
@@ -0,0 +1,60 @@
+# flake8: noqa
+
+""" expose public exceptions & warnings """
+
+from pandas._libs.tslib import OutOfBoundsDatetime
+
+
+class PandasError(Exception):
+    pass
```

Comment:
In particular, I don't think we really want to use `PandasError` going forward, and it would be good to document that.